### PR TITLE
1261 updagre get filtered sc output events for v27

### PR DESCRIPTION
**Pull request by Thykof**
## Delivery Quality Checklist

- [ ] **Breaking Changes in API:**
  Does this PR introduce breaking changes in the API?
    - If yes, have you considered making it backward compatible?
    - If backward compatibility is not considered, set the "breaking-change" label.

- [ ] **Changelog:**
  - [ ] For bugfix PR, set the "bugfix" label
  - [ ] If this change should not appear in changelog, use "ignore-for-changelog" label

- [ ] **Version Update Handling:**
  Have you ensured that the version update by user is handled correctly?

- [ ] **PR Dependency:**
  Does this PR depend on another PR?
    - If yes, is it necessary for the dependency to be released prior to merging this one?

- [ ] **Documentation:**
  - [ ] Are any necessary changes made to user-facing documentation?
  - [ ] Confirm that API documentation is updated with any relevant changes.
  - [ ] Check that README and other documentation files are accurate and current.

### CODE REVIEW

**Comment by Thykof** [comment]
tested on securenetv27:

```
Zipping test website
PUT /websiteUploader/prepare :: Prepare a website for creation.       ....json response: {'code': 'WebCreator-0001', 'message': 'adding DNS record \'testj5xm5cwr\' => \'AS1dQRY4zkwMgCaMECFikH6MCt5dyqqGi1nLhyHoCUkRDgk9xvHz\': calling setResolver with \'[12 0 0 0 116 101 115 116 106 53 120 109 53 99 119 114 52 0 0 0 65 83 49 100 81 82 89 52 122 107 119 77 103 67 97 77 69 67 70 105 107 72 54 77 67 116 53 100 121 113 113 71 105 49 110 76 104 121 72 111 67 85 107 82 68 103 107 57 120 118 72 122 19 0 0 0 100 101 115 99 114 105 112 116 105 111 110 106 53 120 109 53 99 119 114]\' at \'AS1pzpVk79cubdM7Zk9pNdg1JR4qKooVa6usyquiCEQhPeRWD9k7\': listening events for opId at O12DXuaKjtF3DSiPfyy9SrUA5Nv2Dw9WGsUTtSDRTNBhJWs42nCz : {"massa_execution_error":"Runtime error: runtime error when executing operation O12DXuaKjtF3DSiPfyy9SrUA5Nv2Dw9WGsUTtSDRTNBhJWs42nCz: Cache error: Load error: Loading invalid module"}'}
PUT /websiteUploader/prepare :: Prepare a website for creation.       | FAIL |
500 != 200
```

**Comment by Thykof** [comment]
tested on securenet, it works, for webonchain, better put some fees (1000nanoMAS) on the appendBytesToWebsite.

**Review by thomas-senechal — COMMENTED** [review]
Didn't tested but looks good to me
But shouldn't we add secure net to https://github.com/massalabs/station/blob/main/int/config/config_network.yaml ?

File: pkg/node/sendoperation/sendoperation.go [hunk 1]
```diff
@@ -15,7 +15,8 @@ import (
 )
 
 const (
-	DefaultGasLimit            = 700_000_000
+	DefaultGasLimitExecuteSC   = 3_980_167_295
+	DefaultGasLimitCallSC      = 4_294_167_295
```
Comment (peterjah) [inline pkg/node/sendoperation/sendoperation.go:19]: not sure about this.. but we have to think about it more globally

**Review by peterjah — APPROVED** [review]

**Pull request merged**